components/Button/Button.tsx

```diff
@@ -29,7 +29,6 @@ export const border = css`
 `;
 
 const Button = styled.button<ButtonProps>`
-	height: 41px;
```

Comment:
Removing this broke styling on a few other buttons, is there a reason you removed this?

<img width="231" alt="image" src="https://user-images.githubusercontent.com/10401554/167945050-2b3eb02d-253e-4965-bcdf-3a96d81dc431.png">

<img width="302" alt="image" src="https://user-images.githubusercontent.com/10401554/167945080-5cd3d0ca-e4a6-40bf-b023-f3be8e85cf3b.png">

Reply:
Here is the [Discord thread](https://discord.com/channels/852273007370960937/972186157750296597/974018905234358352). The solution I came up was to modify button style in `TabButtonContainer` and removing default height was suggested in Discord discussion. I think eventually we ended up to restore to the initial solution.